tests/regression/cfg/util/dune

```diff
@@ -4,6 +4,7 @@
    goblint-cil
    goblint_logs
    goblint_common
+   goblint_lib ; TODO: avoid
```

Comment:
What's up with the TODO?

Reply:
Previously the `cfgDot` testing utility didn't depend on the entirety of goblint.lib because it's really just concerned with CFGs and thus less unnecessary work by `dune runtest`.

This is now needed because the witness node predicates are in goblint.lib. I'll add a comment here.